source/adiabatic_conditions/compute_profile.cc

```diff
@@ -342,15 +399,18 @@ namespace aspect
 {
   namespace AdiabaticConditions
   {
-    ASPECT_REGISTER_ADIABATIC_CONDITIONS_MODEL(InitialProfile,
-                                               "initial profile",
+    ASPECT_REGISTER_ADIABATIC_CONDITIONS_MODEL(ComputeProfile,
+                                               "compute profile",
                                                "A model in which the adiabatic profile is "
-                                               "calculated once at the start of the model run. "
+                                               "calculated by solving the hydrostatic equations for "
+                                               "pressure and temperature in depth. "
                                                "The gravity is assumed to be in depth direction "
                                                "and the composition is either given by the initial "
                                                "composition at reference points or computed "
                                                "as a reference depth-function. "
                                                "All material parameters are computed by the "
-                                               "material model plugin.")
+                                               "material model plugin. The surface conditions are "
+                                               "either constant or changing over time as prescribed "
+                                               "by an user-provided function.")
```

Comment:
"by a user-provided"?

Reply:
I never consciously thought about these rules, but it makes sense now. Thanks :smile:.

Reply:
Yes, it's not about the letter but how it's pronounced :-)